pallets/parachain-staking/src/reward_rate.rs

```diff
@@ -58,16 +59,23 @@ impl RewardRateInfo {
 		}
 	}
 
-	pub fn compute_collator_reward<T: Config>(&self, issue_number: BalanceOf<T>) -> BalanceOf<T> {
-		self.collator_rate * issue_number
+	pub fn compute_collator_reward<T: Config>(
```

Comment:
Oops, did I forget to remove this from the previous MR? Can you check whether we can remove it or not?